pkg/cli/testdata/MockClient.go

```diff
@@ -382,76 +451,24 @@ func (m *MockClient) VolumeDelete(ctx context.Context, name string) (*apiv1.Volu
 }
 
 func (m *MockClient) ImageList(ctx context.Context) ([]apiv1.Image, error) {
-	return []apiv1.Image{{
```

Comment:
Personally, I am a fan of leaving this in and just adding the `if` check at the beginning. That way, if a user only wants to use a basic list of images, they can do that with minimal effort.

You could also remove the `DefaultImagesList` from the tests in this case. What are your thoughts on this?

Reply:
I was going back and forth on this so, I don't mind adding it back in